index.html

```diff
@@ -1226,6 +1244,52 @@
       <a href="#steps-receiving">Receiving and parsing content</a>
       and <a>Writing or pushing content</a> sections.
     </p>
+    <p>
+      To <dfn>convert an NDEFRecord object's bytes</dfn> to a given <var>type</var>, run these steps:
+    </p>
+    <ol>
+      <li>Let <var>bytes</var> be the <a>NDEFRecord</a> object's associated <a>bytes</a>.
+      </li>
+      <li>Let <var>recordType</var> be the value of <a>NDEFRecord</a> object's recordType attribute.
+      </li>
+      <li>Switch on <var>type</var>:
```

Comment:
I have never seen "switch" used in specs. We usually would say "If type is A, ..., Otherwise, if type is B,..."

@marcoscaceres sorry for needing your input so much on this :-)

Reply:
Here's the example I referenced from: https://www.w3.org/TR/push-api/#dfn-extract-a-byte-sequence

Reply:
I like the switch also, tbh. I've also used:

> If it happens to be the case that X is "foo"

However, your HTML might  now be invalid below... can an`ol` be a child of  `dl`? If no, then wrap the `ol` in a `dd`:

```
<dl>
  <dd><ol><li></dd>
</dl>
```

Reply:
You get the idea :)